### PR TITLE
[MTDSA-428] Improve JSON field descriptions for source of income

### PR DESCRIPTION
**Pull request by dabd**

### CODE REVIEW

File: app/uk/gov/hmrc/selfassessmentapi/domain/selfemployment/SourceType.scala [hunk 1]
```diff
@@ -34,21 +34,27 @@ object SourceType {
...
       FullFieldDescription(name, "commencementDate", "Date", "2016-01-01", "Date in yyyy-dd-mm format"),
       FullFieldDescription(name, "allowances", "Object", "", "Allowances claimed for this self-employment", optional = true),
-      PositiveMonetaryFieldDescription(name, "annualInvestmentAllowance", optional = true),
-      PositiveMonetaryFieldDescription(name, "capitalAllowanceMainPool", optional = true),
-      PositiveMonetaryFieldDescription(name, "capitalAllowanceSpecialRatePool", optional = true),
-      PositiveMonetaryFieldDescription(name, "restrictedCapitalAllowance", optional = true),
-      PositiveMonetaryFieldDescription(name, "businessPremisesRenovationAllowance", optional = true),
-      PositiveMonetaryFieldDescription(name, "enhancedCapitalAllowance", optional = true),
-      PositiveMonetaryFieldDescription(name, "allowancesOnSales", optional = true),
+      PositiveMonetaryFieldDescription(name, "annualInvestmentAllowance",
```
Comment (danmassie) [inline app/uk/gov/hmrc/selfassessmentapi/domain/selfemployment/SourceType.scala]: Should we be mentioning rules for previous tax years?

File: app/uk/gov/hmrc/selfassessmentapi/domain/ukproperty/SourceType.scala [hunk 2]
```diff
@@ -25,21 +25,41 @@ object SourceType {
...
 
     override val fieldDescriptions = Seq(
       FullFieldDescription(name, "allowances", "Object", "", "Allowances claimed for this property", optional = true),
-      PositiveMonetaryFieldDescription(name, "annualInvestmentAllowance", optional = true),
-      PositiveMonetaryFieldDescription(name, "businessPremisesRenovationAllowance", optional = true),
-      PositiveMonetaryFieldDescription(name, "otherCapitalAllowance", optional = true),
-      PositiveMonetaryFieldDescription(name, "wearAndTearAllowance", optional = true),
+      PositiveMonetaryFieldDescription(name, "annualInvestmentAllowance",
+        """You can claim Annual Investment Allowance
+          |(AIA) if you bought equipment (but not cars)
```
Comment (danmassie) [inline app/uk/gov/hmrc/selfassessmentapi/domain/ukproperty/SourceType.scala]: Is the spacing for these multi-line description correct when rendered?

Comment (dabd) [inline app/uk/gov/hmrc/selfassessmentapi/domain/ukproperty/SourceType.scala]: Yes, leading whistespace is removed by stripMargin.

File: app/uk/gov/hmrc/selfassessmentapi/domain/selfemployment/SourceType.scala (hunk 1 above)
Comment (dabd) [inline app/uk/gov/hmrc/selfassessmentapi/domain/selfemployment/SourceType.scala]: Simplified the description.

**Comment by danmassie** [comment]
:+1:

**Pull request merged**